Specs/Core/EllipsoidRhumbLineSpec.js

```diff
@@ -0,0 +1,581 @@
+        var ellipsoid = Ellipsoid.WGS84;
+        var fifteenDegrees = Math.PI / 12.0;
+        var initial = new Cartographic(-CesiumMath.PI + fifteenDegrees, 0.0);
+        var final = new Cartographic(CesiumMath.PI - fifteenDegrees, 0.0);
+
+        var distance = ellipsoid.maximumRadius * 2 * fifteenDegrees;
+
+        var rhumb1 = new EllipsoidRhumbLine(initial, final, ellipsoid);
+        var rhumb2 = new EllipsoidRhumbLine.fromStartHeadingDistance(initial, 3.0 * CesiumMath.PI_OVER_TWO, distance, ellipsoid);
+
+        expect(-CesiumMath.PI_OVER_TWO).toEqualEpsilon(rhumb1.heading, CesiumMath.EPSILON12);
+        expect(CesiumMath.PI / 6 * ellipsoid.maximumRadius).toEqualEpsilon(rhumb1.surfaceDistance, CesiumMath.EPSILON6);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+
+        var rhumb3 = new EllipsoidRhumbLine(final, initial, ellipsoid);
+        var rhumb4 = new EllipsoidRhumbLine.fromStartHeadingDistance(final, CesiumMath.PI_OVER_TWO, distance, ellipsoid);
+        expect(CesiumMath.PI_OVER_TWO).toEqualEpsilon(rhumb3.heading, CesiumMath.EPSILON12);
+        expect(CesiumMath.PI / 6 * ellipsoid.maximumRadius).toEqualEpsilon(rhumb3.surfaceDistance, CesiumMath.EPSILON6);
+        expect(rhumb3.heading).toEqualEpsilon(rhumb4.heading, CesiumMath.EPSILON12);
+        expect(rhumb3.surfaceDistance).toEqualEpsilon(rhumb4.surfaceDistance, CesiumMath.EPSILON6);
+    });
+
+    it('test across equator', function() {
+        var ellipsoid = Ellipsoid.WGS84;
+        var fifteenDegrees = Math.PI / 12.0;
+        var oneDegree = Math.PI / 180.0;
+        var initial = new Cartographic(fifteenDegrees, -oneDegree);
+        var final = new Cartographic(fifteenDegrees, oneDegree);
+
+        //A rhumb line with heading = 0 should be almost the same as a geodesic
+        var rhumb = new EllipsoidRhumbLine(initial, final, ellipsoid);
+        var geodesic = new EllipsoidGeodesic(initial, final, ellipsoid);
+        expect(0.0).toEqualEpsilon(rhumb.heading, CesiumMath.EPSILON12);
+        expect(geodesic.startHeading).toEqualEpsilon(rhumb.heading, CesiumMath.EPSILON12);
+    });
+
+    it('test on equator', function() {
+        var ellipsoid = Ellipsoid.WGS84;
+        var initial = new Cartographic(0.0, 0.0);
+        var final = new Cartographic(CesiumMath.PI - 1, 0.0);
+
+        //A rhumb line on the equator should be the same as a geodesic
+        var rhumb = new EllipsoidRhumbLine(initial, final, ellipsoid);
+        var geodesic = new EllipsoidGeodesic(initial, final, ellipsoid);
+        expect(CesiumMath.PI_OVER_TWO).toEqualEpsilon(rhumb.heading, CesiumMath.EPSILON12);
+        expect(geodesic.startHeading).toEqualEpsilon(rhumb.heading, CesiumMath.EPSILON12);
+        expect(geodesic.surfaceDistance).toEqualEpsilon(rhumb.surfaceDistance, CesiumMath.EPSILON4);
```

Comment:
Same note about distances and geodesics.

Reply:
ah wait nevermind I can't read, it's already checking ~equality between surface distances